src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/config/ControllerAdvice.kt

```diff
@@ -125,8 +145,8 @@ class ControllerAdvice {
       )
   }
 
-  @ExceptionHandler(ValidationException::class, MethodArgumentNotValidException::class)
-  fun handleValidationException(e: ValidationException): ResponseEntity<ErrorResponse> {
+  @ExceptionHandler(ValidationException::class)
+  fun handleValidationException(e: Exception): ResponseEntity<ErrorResponse> {
```

Comment:
This only picks up `ValidationException`, it looks "less wrong" to pass that through

Reply:
We've been bitten by bugs in this area so I hone in on inconsistency here 😅

Reply:
makes sense